mdal/frmts/mdal_xmdf.cpp

```diff
@@ -117,6 +119,27 @@ bool MDAL::DriverXmdf::canReadDatasets( const std::string &uri )
   return true;
 }
 
+
+void MDAL::DriverXmdf::readGroupsTree( HdfFile &file, const std::string &name, MDAL::DatasetGroups &groups, size_t vertexCount, size_t faceCount ) const
+{
+  HdfGroup gMesh = file.group( name );
+  for ( const std::string &groupName : gMesh.groups() )
+  {
+    HdfGroup gGroup = gMesh.group( groupName );
+    if ( gGroup.isValid() )
+    {
+      if ( groupName == "Maximums" )
```

Comment:
whatabout Minimums?

Reply:
this part come from the former implementation... I guess there is no "Minimums" in TUFLOW file (not in test files).